docs/docs/usage.md

```diff
@@ -1420,6 +1420,31 @@ tasks:
     - ./app{{exeExt}} -h localhost -p 8080
 ```
 
+## `set` and `shopt`
+
+It's possible to specify options to the
+[`set`](https://www.gnu.org/software/bash/manual/html_node/The-Set-Builtin.html)
+and [`shopt`](https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html)
+builtins. This can be added at global, task or command level.
+
+```yaml
+version: '2'
```

Comment:
@pd93 I think that this is an error and should be instead: 
```
version: '3'
```
?

Reply:
@marco-m-pix4d good spot! yes, this should be `version: '3'`. I'll get this updated.